posthog/urls.py

```diff
@@ -36,8 +37,15 @@
     extend_api_router(router, projects_router=projects_router, project_dashboards_router=project_dashboards_router)
 
 
+@csrf.ensure_csrf_cookie
 def home(request, *args, **kwargs):
-    return render_template("index.html", request)
+
+    response = render_template("index.html", request)
+
+    # Delete csrftoken cookie if it exists to reduce cookie size
+    # See https://github.com/PostHog/posthog/pull/8546 for context
+    response.delete_cookie("csrftoken")
```

Comment:
Will this remove the other sites cookie? What happens in this case:

 1. user navigates to the domain.com, csrftoken set for Domain=.domain.com
 2. user opens another tab to posthog.domain.com , csrftoken deleted
 3. user interacts with other site

Is the cookie size a big issue? What would be the effect of leaving this out?

Reply:
Ah yep 🙈 . Probably better to keep it in.

Cookie size has been an issue historically with posthog-js so I am hesitant in general, but hopefully should be fine for the app.